common/djangoapps/course_modes/models.py

```diff
@@ -30,6 +30,9 @@ class CourseMode(models.Model):
     We would like to offer a course in a variety of modes.
 
     """
+    class Meta:
+        app_label = "course_modes"
+
```

Comment:
I'm not really clear on why this is necessary. [The Django docs](https://docs.djangoproject.com/en/1.8/ref/models/options/#app-label) seem to say that it's only needed when declaring a model outside of an application in `INSTALLED_APPS`, and it looks to me like `course_modes` is included in `lms/envs/common.py` and `cms/envs/common.py`.